tools/erc721tester/erc721tester.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package main
+
+import (
+	"context"
+	"github.com/iotexproject/iotex-core/tools/erc721tester/blockchain"
+	"math/big"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/iotexproject/iotex-core/address"
+	"github.com/iotexproject/iotex-core/blockchain/block"
+	"github.com/iotexproject/iotex-core/config"
+	"github.com/iotexproject/iotex-core/pkg/keypair"
+	"github.com/iotexproject/iotex-core/pkg/log"
+	"github.com/iotexproject/iotex-core/pkg/probe"
+	"github.com/iotexproject/iotex-core/server/itx"
+	"github.com/iotexproject/iotex-core/testutil"
+	"github.com/iotexproject/iotex-core/tools/erc721tester/assetcontract"
+)
+
+func main() {
```

Comment:
Merge this test into "executiontester.go", then the test structure looks like this:
1. start a single IoTeX server; (shared by erc721_token and fp_token);
2. create two accounts; (shared by erc721_token and fp_token);
3. create two (three if taking stable token into account) token contracts (prerequisite deployment should be done in function StartContracts, then invoke CreateToken respectively);
4. transfer fp token between the two accounts;
5. transfer erc721 token between the two account;
5. compare the actual sum of fp tokens with the amount of creation;
6. compare the actual sum of erc721 tokens with the amount of creation.